test/Auto_test/ptest.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+source ./shared-queens.rc
+openstack token issue
+# get token
```

Comment:
You needn't to issue a token at first, right?

Reply:
Yes, I have fixed it.